CHANGES.txt

```diff
@@ -0,0 +1 @@
+1.0.0 - First version of the plugin
```

Comment:
Maybe we keep plugins version 0.0.x until we have a first official release of the pluginization and we bump it to 1.0.0?

Reply:
I think first plugins stable version make sense to be 1.0.0--so we should start with less. ANd, yes bump it when happy with it....we have soon to modify buildbot to understand new structure, test each plugin and report back to, ideally and specific slack channel.